app/public/cantusdata/fixtures/2_initial_data.json

```diff
@@ -37,19 +37,6 @@
       "model": "cantusdata.manuscript",
       "pk": 3
     },
-    {
```

Comment:
Curious - why does CU use fixtures, and why are they committed to the repo?

Reply:
Right now, our test database (ie. the one against which our test suite is run) is populated by these fixtures.

Reply:
ah - got it. that makes sense.